README.md

```diff
@@ -0,0 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Hello Microverse project is introduction to well documented project in professional way
+
+![screenshot](./webpage_screenshot.png)
+
+Porject's features are added into seperate branch to keep main branch safe.
+
+## Built With
+
+- HTML and CSS
+- Github and Git
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+Live Demo link will be updated after this project is approved by Code Reviewer Tseam
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Add details on how to download the repo and how you can go to the file and open it.